usaspending_api/search/v2/views/spending_by_category.py

```diff
@@ -0,0 +1,205 @@
+import logging
+
+from django.db.models import F, Sum
+
+# from usaspending_api.awards.v2.filters.sub_award import subaward_filter
+from usaspending_api.awards.v2.filters.view_selector import spending_by_category as sbc_view_queryset
+from usaspending_api.common.exceptions import InvalidParameterException
+from usaspending_api.common.helpers import get_simple_pagination_metadata
+from usaspending_api.core.validator.award_filter import AWARD_FILTER
+from usaspending_api.references.models import Cfda
+
+logger = logging.getLogger(__name__)
+
+
+class business_logic:
+    __slots__ = ('category', 'scope', 'page', 'limit', 'lower_limit', 'upper_limit', 'filters', 'queryset', 'model', )
+
+    def __new__(cls, payload):
+        """
```

Comment:
Note from python's documentation: 
```
__new__() is intended mainly to allow subclasses of immutable types (like int, str, or tuple) to customize instance creation. It is also commonly overridden in custom metaclasses in order to customize class creation.
```
I'm not finding a strong use case for using `__new__` over `__init__`. Need more clarification in comment.